styles.css

```diff
@@ -1,16 +1,62 @@
+body {
+  min-height: 100vh;
+  position: relative;
+}
+
+/* Nav-bar */
+header {
+  padding-top: 5px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px 20px;
+  border: 2px #000 solid;
+  margin: 0 10px 0 10px;
+}
+
+.links-list {
+  list-style: none;
+  display: flex;
+  gap: 20px;
+  align-items: center;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+}
+
+.nav-link.active {
+  color: orangered;
```

Comment:
- [ ] Good design setup :+1:  But there is a few different between your design and the wiframe provided for this project. Please make sure that your design match to the wireframe below . As a developer it very important to pay attention to details and also follow the project requirement to make your app or website works as expected :+1: 

![awesome_books_full_website](https://user-images.githubusercontent.com/6615394/206569314-668165a5-5189-4a53-b4d4-283cc76a97fc.png)

Reply:
@codecaiine I MAKE THE WEBSITE ACCORDING TO WIREFRAME CAN YOU CHAEK NOW